src/Compute/Compute/Manual/VirtualMachineScaleSetCreateOrUpdateMethod.cs

```diff
@@ -180,6 +180,11 @@ public partial class NewAzureRmVmss : ComputeAutomationBaseCmdlet
         [Parameter(ParameterSetName = SimpleParameterSet, Mandatory = false)]
         public SwitchParameter EncryptionAtHost { get; set; }
 
+        [Parameter(ParameterSetName = SimpleParameterSet, Mandatory = false,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Fault Domain count for each placement group.")]
+        public int? PlatformFaultDomainCount { get; set; }
```

Comment:
i believe '?' is not necessary here to be able to pass this parameter as null.  Should behave the same when the variable type is 'int',   check 'maxprice' parameter. I

Reply:
@grizzlytheodore , having the parameter as the nullable int type (int?) removes the need to cast it as a nullable int in the resourceGroup.CreateVirtualMachineScaleSetConfig() call. 
MaxPrice does have this cast (double?), so having the parameter int? removes the need for this cast.

Reply:
@Sandido  I see. I think it's better to stay consistent on where the parameter is being casted as a nullable variable. Not only in this cmdlet, the variables are casted as a nullable variable in CreateVirtualMachineScaleSetConfig(), but other cmdlets follow the similar structure where '?' is left out at the parameter declaration, and later is casted as nullable when it is being checked with isParameter() method.

Reply:
@grizzlytheodore , I changed it as you suggested.